src/components/PopoverContent/PopoverContent.module.css

```
@@ -0,0 +1,138 @@
+/*------------------------------------*\
+    # POPOVER CONTENT
+\*------------------------------------*/
+
+/**
+ * A floating panel component with arbitrary content.
+ * Styling for the Popover component since the parent <Popover> component provides functionality.
+ */
+.popover-content {
+  z-index: var(--eds-z-index-top);
+}
+
+/**
+ * Main popover content, as opposed to the visual indicator arrow.
+ */
+.popover-content__main {
+  border-radius: var(--eds-border-radius-md);
+  overflow: auto;
+  box-shadow: var(--eds-box-shadow-md);
+  background-color: var(--eds-theme-color-background-neutral-default);
+}
+
+/**
+ * Decorative arrow pointing the Popover to the trigger element.
+ */
+.popover-content__arrow {
+  position: absolute;
+  width: 0;
+  height: 0;
+}
+
+/**
+ * Arrow directions, made into mixins for brevity in use.
+ */
+@define-mixin down-arrow {
+  border-top: var(--eds-size-half) solid
+    var(--eds-theme-color-background-neutral-default);
+  border-left: var(--eds-size-half) solid transparent;
+  border-right: var(--eds-size-half) solid transparent;
```

I wonder if a clearer way to define these styles could be to define the default border as transparent and then override the other two.

```suggestion
  border: var(--eds-size-half) solid transparent;
  border-top-color: var(--eds-theme-color-background-neutral-default);
  border-bottom: none;
```

I think I mostly find it hard to read because of the long color variable though.

---

Yea that makes sense to me